### DISCUSSION TITLE
Ajouter des "key items", obtenable seulement avec certaines fins d'events et qui en dévérouillent d'autres

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#1299] [Nouvelles idées]
En lien avec #627 et #633

J'aurais imaginé une nouvelle catégorie d'items, pas dropables, juste à collecter pour avoir toutes les fins d'event
En plus ça récompenserait le temps de jeu

**Comment by Pietagorh** [comment]
On les obtiendrait seulement sur certaines fins spéciales grâce à #633 du coup

**Reply by BastLast** [reply]
C'était déjà prévu

**Reply by Pietagorh** [reply]
A oké
